tap_github/sync.py

```diff
@@ -24,10 +24,52 @@ def get_selected_streams(catalog):
 
     return selected_streams
 
+def update_currently_syncing(state, stream_name):
+    """
+    Updates currently syncing stream in the state.
+    """
+    if stream_name is None and ('currently_syncing' in state):
```

Comment:
As you used `set_currently_syncing` below can we use `get_currently_syncing` as well?

Reply:
Updated the code